PhysicsTools/NanoAOD/python/nano_cff.py

```diff
@@ -253,10 +253,12 @@ def nanoAOD_activateVID(process):
     for modname in electron_id_modules_WorkingPoints_nanoAOD.modules:
         setupAllVIDIdsInModule(process,modname,setupVIDElectronSelection)
     process.electronSequence.insert(process.electronSequence.index(process.bitmapVIDForEle),process.egmGsfElectronIDSequence)
-    for modifier in run2_egamma_2017,run2_egamma_2018,run2_miniAOD_80XLegacy,run2_nanoAOD_94XMiniAODv1,run2_nanoAOD_94XMiniAODv2,run2_nanoAOD_94X2016,run2_nanoAOD_102Xv1,run2_nanoAOD_106Xv1:
+    for modifier in run2_egamma_2016,run2_egamma_2017,run2_egamma_2018,run2_miniAOD_80XLegacy,run2_nanoAOD_94XMiniAODv1,run2_nanoAOD_94XMiniAODv2,run2_nanoAOD_94X2016,run2_nanoAOD_102Xv1,run2_nanoAOD_106Xv1:
         modifier.toModify(process.electronMVAValueMapProducer, src = "slimmedElectronsUpdated")
```

Comment:
```suggestion
        (modifier & ~run2_nanoAOD_106Xv2).toModify(process.electronMVAValueMapProducer, src = "slimmedElectronsUpdated")
```
will not need to unset a few lines later

Reply:
it's enough to remove the 'run2_egamma_2016,run2_egamma_2017,run2_egamma_2018' from the above line 256. so that modifier loop over the list of all previous but run2_nanoAOD_106Xv2